### DISCUSSION TITLE
Gitlab oath2 integration

### DISCUSSION THREAD

**Discussion by wallhided** [kafbat/kafka-ui#411] [Q&A] [answered]
Hey! I'm looking for configuration of oauth for self-hosted Gitlab. Does kafka-ui support it? Could you please provide an example of configuration?

**Comment by Haarolean** [comment, marked as answer]
Hi, it's supported as basically any other provider which supports OAuth.

Please take a look at this docs piece: https://ui.docs.kafbat.io/configuration/authentication/oauth2

**Reply by Haarolean** [reply]
A config example could also be found [here](https://github.com/kafbat/kafka-ui/issues/408)
